src/routes/(pages)/app/(app)/[accountId]/+page.svelte

```diff
@@ -47,99 +51,142 @@
-    <YourDripListsSection
-      collapsable
-      collapsed={[
-        ...data.profileData.votingRounds,
-        ...mapFilterUndefined(filterCurrentChainData(data.profileData.chainData).dripLists, (v) =>
+        </header>
+      </SectionSkeleton>
+    </div>
+    <div class="article-content gap-16">
+      <Developer accountId={data.profileData.account.accountId} />
+      <YourProjectsSection
+        collapsable
+        collapsed={mapFilterUndefined([profileChainData], (v) => (v === null ? undefined : v))
+          .length === 0}
+        projects={mapFilterUndefined(profileChainData.projects, (v) =>
+          v === null ? undefined : v,
+        )}
+        withClaimProjectButton={true}
+      />
+      <YourDripListsSection
+        collapsable
+        collapsed={[
+          ...data.profileData.votingRounds,
+          ...mapFilterUndefined(profileChainData.dripLists, (v) => (v === null ? undefined : v)),
+        ].length === 0}
+        votingRounds={data.profileData.votingRounds}
+        dripLists={mapFilterUndefined(profileChainData.dripLists, (v) =>
           v === null ? undefined : v,
-        ),
-      ].length === 0}
-      votingRounds={data.profileData.votingRounds}
-      dripLists={mapFilterUndefined(
-        filterCurrentChainData(data.profileData.chainData).dripLists,
-        (v) => (v === null ? undefined : v),
-      )}
-    />
-    <Supporters
-      collapsable
-      collapsed={filterCurrentChainData(data.profileData.chainData).support.length === 0}
-      type="address"
-      supportItems={filterCurrentChainData(data.profileData.chainData).support}
-    />
-    <Streams
-      hideIncoming
-      collapsable
-      collapsed={filterCurrentChainData(data.profileData.chainData).streams.outgoing.length === 0}
-      userStreams={filterCurrentChainData(data.profileData.chainData).streams}
-      disableActions={!isSelf}
-      accountId={data.profileData.account.accountId}
-    />
-    <Balances
-      collapsable
-      collapsed={filterCurrentChainData(data.profileData.chainData).balances.length === 0}
-      userBalances={filterCurrentChainData(data.profileData.chainData).balances}
-      accountId={data.profileData.account.accountId}
-    />
+        )}
+        withCreateButton={true}
+      />
+      <Supporters
+        collapsable
+        collapsed={profileChainData.support.length === 0}
+        type="address"
+        supportItems={profileChainData.support}
+      />
+      <Streams
+        hideIncoming
+        collapsable
+        collapsed={profileChainData.streams.outgoing.length === 0}
+        userStreams={profileChainData.streams}
+        disableActions={!isSelf}
+        accountId={data.profileData.account.accountId}
+      />
+      <Balances
+        collapsable
+        collapsed={profileChainData.balances.length === 0}
+        userBalances={profileChainData.balances}
+        accountId={data.profileData.account.accountId}
+      />
+    </div>
+    <aside class="article-sidebar">
```

Comment:
do i understand correctly that this would be shown even if orcid is `false` for the current network? shouldn't show this then.